src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -11,15 +11,25 @@
 import static seedu.address.testutil.TypicalPersons.ALICE;
 import static seedu.address.testutil.TypicalPersons.BOB;
 
+import java.util.List;
 import java.util.Set;
 
 import org.junit.jupiter.api.Test;
 
+import seedu.address.model.group.Group;
 import seedu.address.model.tag.Tag;
 import seedu.address.testutil.PersonBuilder;
 
 public class PersonTest {
 
+    @Test
+    public void addGroups_addsGroupSuccessfully() {
+        Person actualPerson = new PersonBuilder().build();
+        Group group = new Group("StudyGroup", List.of(actualPerson));
```

Comment:
Since we will be using this alot for test cases, maybe we can abstract this out? similar to how theres the const ALICE and BOB Person objects for testing usage as well